docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +255,216 @@ You should receive a response indicating there were no errors.
+// TEST[continued]
+<1> This field is dynamically added to the document.
+View the mapping for the `books` index. The new field `new_field` has been added to the mapping with a `text` data type.
+
+[source,console]
+----
+GET /books/_mapping
+----
+// TEST[continued]
+
+.Expand to see example response
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "books": {
+    "mappings": {
+      "properties": {
+        "author": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "name": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "new_field": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "page_count": {
+          "type": "long"
+        },
+        "release_date": {
+          "type": "date"
+        }
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+===============
+
+[discrete]
+[[getting-started-explicit-mapping]]
+==== Define explicit mapping
+
+Create an index named `my-explicit-mappings-books` with explicit mappings:
+
+[source,console]
+----
+PUT /my-explicit-mappings-books
+{
+  "mappings": {
+    "dynamic": false,  <1>
+    "properties": {  <2>
+      "name": { "type": "text" },  <3>
+      "author": { "type": "text" },
+      "release_date": { "type": "date", "format": "yyyy-MM-dd" },  <4>
+      "page_count": { "type": "integer" }  <5>
+    }
+  }
+}
+----
+// TEST[continued]
+<1> Disables dynamic mapping for the index. Documents containing fields not defined in the mapping will be rejected.
+<2> The `properties` object defines the fields and their data types for documents in this index.
+<3> Defines the "name" field as type "text", suitable for full-text search on book titles.
+<4> Specifies the "release_date" field as a "date" type with the specific format `yyyy-MM-dd`.
+<5> Sets the "page_count" field as an "integer" type for storing the number of pages.
+
+.Expand to see example response
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "acknowledged": true,
+  "shards_acknowledged": true,
+  "index": "my-explicit-mappings-books"
+}
+----
+// TEST[skip:already created in setup]
+===============
+
+[discrete]
+[[getting-started-combined-mapping]]
+==== Combine dynamic and explicit mappings
 
-Indexed documents are available for search in near real-time.
+Explicit mappings are defined at index creation, and documents must conform to these mappings.
+When an index has the `dynamic` flag set to `true`, you can add new fields to documents without updating the mapping.
+This allows you to <<mapping-combine-explicit-dynamic,combine explicit and dynamic mappings>>.
```

Comment:
added in https://github.com/elastic/elasticsearch/pull/113558/commits/c0eba9fcafe7b01d0321ce6a732782752b269b33